app/experimenter/nimbus-ui/src/components/ChangeApprovalOperations/index.test.tsx

```diff
@@ -58,7 +59,7 @@ describe("ChangeApprovalOperations", () => {
       "open-remote-settings",
     );
     fireEvent.click(openRemoteSettingsButton);
```

Comment:
I don't think this fireEvent is useful anymore and could probably be removed as you've done elsewhere